iceberg-light.toml

```diff
@@ -0,0 +1,333 @@
+name = "iceberg-light"
+background = "light"
+
+[hues]
+hue_base = 230
+
+hue_red    = 340
+hue_orange = 25
+hue_green  = 90
+hue_cyan   = 200
+hue_blue   = 220
+hue_purple = 260
+hue_pale   = 235
+
+[colors]
+# palette
+blue   = "hsl($hue_blue, 0.55, 0.40)"
+green  = "hsl($hue_green, 0.40, 0.40)"
+cyan   = "hsl($hue_cyan, 0.45, 0.45)"
+orange = "hsl($hue_orange, 0.55, 0.50)"
+purple = "hsl($hue_purple, 0.38, 0.53)"
+red    = "hsl($hue_red, 0.55, 0.56)"
+pale   = "hsl($hue_pale, 0.30, 0.45)"
+
+# normal
+normal_bg = "hsl($hue_base, 0.10, 0.92)"
+normal_fg = "hsl($hue_base, 0.20, 0.25)"
+
+# TODO experimental colors
+pink = "hsl(330, 0.32, 0.68)"
+teal = "hsl(165, 0.32, 0.65)"
+
```

Comment:
These were unused experimental colors of the dark variant so they are redundant here

Reply:
gotcha, just removed these